chain/tron/core/README.md

```diff
@@ -0,0 +1,3 @@
+## Generation
+
+Proto files were generated from the [TRON protocol repository](https://github.com/tronprotocol/protocol) at commit [37bb922a9](https://github.com/tronprotocol/protocol/commit/37bb922a9).
```

Comment:
Can you include a makefile/justfile/script that does the generation?  So if they need to be updated, the script can be used.